### DISCUSSION TITLE
Xenomorph revival project of 2022

### DISCUSSION THREAD

**Discussion by Edan52** [ParadiseSS13/Paradise#18888] [Projects]
While terror spiders are cool and all, them being the only major infestation threat, and only midround besides blob that can really take over the station is not great, doubly so when those two both have their shortcomings.

Xenomorphs have had a couple changes recently to make them more viable as a midround, I personally don't really think they need completely redoing to be in the game again, but they still have major issues that really need addressing. The goal is to make some changes to improve the balance and overall gameplay experience of xenomorphs, with the goal being to test these changes once at least a couple have been implemented, and ultimately re-adding them as a mid-round antag.

A lot has changed since the time they were removed, for example security borgs are no longer a balance concern, same with the tesla revolver. Lethal shotguns not being an option unless you get gamma is also something to consider (i'm aware of their burn weakness). Mechs also no longer instantly sleep xenomorphs.

Here is a list of the major issues we came up with so far:

- Facehuggers are too binary, they are either a game over or completely nullified by being blocked with the correct equipment.
- Neurotoxin spit needs a cooldown.
- Neurotoxin split needs adjusting to work with stamina-based combat
- Abilities need to at the very least be actions, tab combat abilities these days are a no.
- Corrosive acid is a literal one shot ability, and the select target list is horrible to use.
- Using disarm as a xeno after the tweak is a bit too weak.

Can add more if people come up with anything. Considering that list, here are the actions:

- [x] Give neurotoxin spit a cooldown. #18919
- [x] Adjust neurotoxin spit to work for stamina based combat #18919
- [ ] Let the player decide which direction neurotoxin spit fires from
- [ ] Make corrosive acid not one shot people without clothes, or any simple mob with less than 400 health
- [ ] Slightly buff the disarm on xenomorphs (my initial recommendation is to make it deal 40 stamina damage per hit, and knocking down people with over 50 stamina damage)
- [x] Make facehuggers less of a binary tool that can either game end somebody instantly or is completely mitigated by the correct equipment (please discuss before tackling this one). #19025
- [ ] Move the abilities from tab to proper actions with icons.
- [ ] Give the sentinel stalk a function.
- [ ] Make their abilities use spells or action datums.

These are the maybes:

- [ ] Add some sort of extra counter to mechs (corrosive acid disables their weapons for a bit maybe?)

Ideally message the balance or design team on discord with any questions and ping me (Edanski#1666) if you plan to take on any of these, so I can put your name next to it and not have multiple people working on something at once!

**Comment by GDNgit** [comment]
dibs on 
`Let the player decide which direction neurotoxin spit fires from`, `Move the abilities from tab to proper actions with icons.`,
 `Make their abilities use spells or action datums.`
through #20002
